generator/internal/codegen_utils.cc

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "generator/internal/codegen_utils.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_join.h"
+#include "absl/strings/str_replace.h"
+#include "absl/strings/str_split.h"
+#include <cctype>
+#include <string>
+
+namespace google {
+namespace cloud {
+namespace generator_internal {
+
+namespace {
+char const kGeneratorFileSuffix[] = ".gcpcxx.pb";
+}  // namespace
+
+std::string GeneratedFileSuffix() { return {kGeneratorFileSuffix}; }
```

Comment:
There's no need to declare the string as a constant, since it's only used in exactly one place. Here this function can simply `return ".gcpcxx.pb";`.

Reply:
Done.